rst/cartridge_admin.rst

```diff
@@ -718,12 +834,20 @@ Connecting to nodes via CLI
 Each Tarantool node (``router``/``storage``) provides an administrative console
 (Command Line Interface) for debugging, monitoring, and troubleshooting. The
 console acts as a Lua interpreter and displays the result in the human-readable
-YAML format. To connect to a Tarantool instance via the console, say:
+YAML format. To connect to a Tarantool instance via the console, you can choose
+one of the commands:
 
 ..  code-block:: bash
 
+    # old-fashioned way
```

Comment:
Let's split this code block to a bullet list with three items <description>: <command>. 
Keeping three independent options in a single block is not good because it can't be used as a whole.